h/static/images/icons/google-plus.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="22px" height="14px" viewBox="0 0 22 14" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 41.2 (35397) - http://www.bohemiancoding.com/sketch -->
+    <title>logo_google_plus_32px_clr copy 6</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="logo_google_plus_32px_clr-copy-6" fill="#A6A6A6">
```

Comment:
For the other monochrome SVG icons, I've replaced references to a hard-coded fill color with `currentColor`, so that the icon inherits its color from the CSS `color` property of the containing `<svg>` element.